llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp

```diff
@@ -2693,33 +2677,6 @@ void DebugLocEntry::finalize(const AsmPrinter &AP,
     List.setTagOffset(*DwarfExpr.TagOffset);
 }
 
-inline static DbgValueLoc mkDbgValueLoc(const DIExpression *expr,
```

Comment:
Again, can you write some words why this had to go?

Reply:
This change comes from the original code https://github.com/llvm/llvm-project/blob/release/13.x/llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp#:~:text=List.setTagOffset(*DwarfExpr.TagOffset)%3B

Reply:
Paul commented offline that this change comes from release_12x, I see that the issue was that the base code for release/13.x is different and when I applied the commit for this piece of code the build was breaking, hence I remove the code. What I will do now is to pick up the commit again and do the required changes for release_13x be compatible with it. I will commit-push again once is ready.